### DISCUSSION TITLE
[스케줄 및 방향 조율] 스터디 Reboot

### DISCUSSION THREAD

**Discussion by yujin1292** [yujin1292/BreakingCodingTest#36] [General]
각자의 스케줄 변동으로 인해, 기존 방식 ( 주 1회 Google Meet 으로 진행 ) 과는 다르게 새로운 방식을 도입하고자 합니다!

두시간 가량의 수업 후 숙제 진행에서 아래와 같은 방법으로 변경하고자 하는데 
선택지를 고르거나 더 좋은 의견 있으면 댓글로 남겨주세요
@j-miiin @jeongiin 

---

### Option 1

1. 스터디 멘토 ( @yujin1292 ) 가 공부할 주제를 선정 ( 주제 당 2주 이상 소요 예상 )
2. 선정한 주제를 공부 하고, 공부 내용을 README.md 파일로 작성 ( 1주 소요 )
3. 선정한 주제와 관련된 문제를 멘토가 선정, 기존방식처럼 풀고 파일 제출 및 코드리뷰 
    - 해당 주제에서 다룰 문제가 많다면 1주 이상 진행

### Option 2
1. 스터디 멘토 ( @yujin1292 ) 가 공부할 주제를 선정 하고 각자 공부 ( 주제 당 2주 이상 소요 예상 )
    - 공부 한 내역은 Repo에 올려도 괜찮고 안 올려도 괜찮음, 따로 공부하는 시간은 주지 않고 각자 진행
2. 선정한 주제와 관련된 문제를 멘토가 각 멘티별로 다르게 선정, 기존 방식처럼 풀고 파일 제출 및 코드리뷰 
    - 코드 리뷰는 본인을 제외한 멘토, 멘티가 모두 리뷰 해야 하고, 남이 봤을 때 이해 할 수있도록 친절한 주석 달기
3. 다른 멘티가 풀었던 문제를 바꿔서 풀어보고 코드리뷰 진행  
    - 코드 리뷰는 멘토 만 진행

**Comment by jeongiin** [comment]
먼저 일정 조율해가면서 스터디 이끌어나가주시는 멘토님께 감사의 말씀을 드립니다•••

1번 방법이 깔끔하고 좋다고 생각합니다!!!
1번으로 하더라도 주석은 더 꼼꼼하게 달아볼게요!!

**Comment by j-miiin** [comment]
저도 1번 방법이 좋을거 같습니다!!!

**Comment by yujin1292** [comment]
그럼 1번 방법으로 진행할게요!
상세 계획과 일정은 내일 공지 및 업로드 하겠습니다~~
@jeongiin @j-miiin
